citi-onboarding/Delta-odonto
# Feature/banner front

## WHAT I DID
Created the front-end of the banner

## HOW TO TEST

Follow the steps
 1 - Open the folder client typing this command on the terminal:
´cd client´

2-  Then type this command on the terminal:
´yarn start´

3- After this acess [http://localhost:3000](url) on your browser

You will me able to see the front-end of the banner section.
It contains:
 a title,
 a description,
 our services button,
a background image of a dentist,
 a logo and a arrow button (only if you are acessing from a mobile screen).
You can also see the mockup if you acess this link [https://www.figma.com/file/WO1wTZDVcs145XzB3HRY5M/Odonto-OnePage?node-id=24%3A2](url)
